tabcmd/commands/commands.py

```diff
@@ -55,7 +55,27 @@ def get_data_source_id(server, datasource_name):
     @staticmethod
     def exit_with_error(logger, message, exception=None):
         if exception:
-            logger.error(message, exception)
+            Commands.check_common_error_codes(logger, exception)
         else:
             logger.error(message)
         sys.exit(1)
+
+    @staticmethod
+    def check_common_error_codes(logger, error):
+        if error.code.find("404") == 0:
+            logger.debug("Not Found: Resource cannot not be located")
```

Comment:
what's the benefit of having both lines? Shouldn't the error line show up in the debug log too?

Reply:
Might also be worth printing the complete code, if this is the 6 digit one?

Reply:
Thanks, Fixed. I thought for some reason errors were printed to the console and debug strings went to the log.